virt/virt-2-4-release-notes.adoc

```diff
@@ -102,7 +102,7 @@ from being inadvertently started before an upload has completed.
 //Placeholder for new content.
 
 //CNV-5465-Disconnected installation
-* {VirtProductName} can be installed on disconnected clusters in restricted network environments that do not have Internet connectivity. You can create local mirrors for the {VirtProductName} Operator, and install the Operator from a local catalog image that is accessible to the disconnected cluster. Learn more about xref:../operators/olm-restricted-networks.adoc#olm-understanding-operator-catalog-images_olm-restricted-networks[using Operator Lifecycle Manager on restricted networks].
+* {VirtProductName} can be installed on disconnected clusters in restricted network environments that do not have Internet connectivity. You can create local mirrors for the {VirtProductName} Operator, and install the Operator from a local catalog image that is accessible to the disconnected cluster. Learn more about xref:https://access.redhat.com/articles/4827691[Installing OpenShift Virtualization on a restricted network cluster].
```

Comment:
We're linking to external content so this needs to be a 'link'.
ie:  Learn more about link:https://access.redhat.com/articles/4827691[Installing OpenShift Virtualization on a restricted network cluster].

Reply:
+1, this should fix that build error.

Reply:
Done.